src/robusta/utils/common.py

```diff
@@ -37,3 +38,14 @@ def duplicate_without_fields(obj: HikaruBase, omitted_fields: List[str]):
             pass  # in case the field doesn't exist on this object
 
     return duplication
+
+
+def to_pascal_case(s: str):
+    """Convert to PascalCase by capitalizing each word and removing spaces"""
+
+    if not s:
+        return s
+
+    words = re.split(r'\W+', s)
+
+    return ''.join(word.capitalize() for word in words if word.isalnum())
```

Comment:
what happens is `word.isalnum()` is false?
will it change the provided value? (i.e. drop some word)

Why do we need this check?

I'm not sure we need this function. It's only used in 2 places, which I'm not sure are a must